### ISSUE TITLE
font-weight-normal utility doesn't work in o-footer

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#58]
Because of `.o-footer * { font-weight: bold; }`, the `font-weight-normal` utility doesn't apply in the Orange footer component.

This means I need to patch:
```css
.o-footer {
  .font-weight-normal {
    font-weight: 400;
  }
}
```

But as a utility class, it might be better to simply use `!important`.

**Comment by ygatesoupe** [comment]
Fixed in commit 19ab16513fc6 so class `.font-weight-normal` can be used on links and text.

Keep in mind that according to Brand all text in footer shall be bold.

**Comment by ffoodd** [comment]
Oh I know this. The graphic designer doesn't care, though :)
Thanks for fixing!
